src/components/DonationWidget/components/DonationLoading.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as DonationWizard from './DonationWizard';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface Props {}
+
+const DonationThankYou: React.FC<Props> = () => {
```

Comment:
```suggestion
const DonationThankYou = () => {
```

Reply:
If you were removed the interface I would have been able to commit from here _(I wanted to use that feature)_